src/app/backend/resource/deployment/rollout.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2017 The Kubernetes Authors.
```

Comment:
Why is it named `rollout.go`?

s/rollout/rollback

Reply:
This is because it not only has rollback feature, but also pause and resume feature. These functions are defined as` kubctl rollout pause / resume / history XXX` in kubernets, and all use the `kubectl rollout` command. In this file, I also implement these functions. And maybe I should also write some endpoint interfaces for them.

Reply:
Ok, `rollout` can stay then.